recipes/spacepy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: spacepy
+  version: "0.1.6"
+
+source:
+  fn: spacepy-0.1.6.tar.gz
+  url: https://pypi.python.org/packages/f7/10/4a9a43014d25c1ce08aad136da2515eb120a26105dec5e8f42b5986c22a3/spacepy-0.1.6.tar.gz
+  md5: 1272ec6537ca45b6e4b8ea3f8c2cf8a1
+
+build:
+  number: 0
+  features:
+    - vc9      [win and py27]
+    - vc10     [win and py34]
+    - vc14     [win and py35]
+
+requirements:
+  build:
+    - python
+    - numpy
+    - gcc      [not win]
+
+  run:
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - python-dateutil
+    - h5py
+    - libgfortran   [not win]
+    - libgcc        [not win]
+
+test:
+  imports:
+    - spacepy
+    - spacepy.irbempy
+    - spacepy.plot
+    - spacepy.pybats
+    - spacepy.pycdf
+    - spacepy.toolbox
+
+about:
+  home: http://spacepy.lanl.gov
```

Comment:
That's because we're being dinged by annual classification review and I'm having a hard time twisting arms to get it done. That is the homepage; the pythonhosted is just an occasionally-updated mirror.